src/components/bookappointment/modal/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { Close } from '@mui/icons-material';
+import {
+  CssBaseline,
+  Container, Paper, IconButton, Typography, Stepper, Step, StepLabel, Box, Button, Modal,
+} from '@mui/material';
+import * as React from 'react';
+import { FormikProps, useFormik } from 'formik';
+import ModelStyle from '../../../pages/therapistProfile/classes';
+import BookAppointment from '../selectTime/index';

```

Comment:
no need to type index here
```suggestion
import BookAppointment from '../selectTime';

```